### ISSUE TITLE
Doesn't handle certain class names

### ISSUE THREAD

**Issue by necolas** [webpack/css-loader#72]
It should respect escaped characters in CSS.

Source:

```css
/* can be used in html as "some:selector" */
.some\:selector { ... }
```

Expected:

```css
._aHash { ... }
```

Actual:

```css
._aHash\:selector { ... }
```

**Comment by necolas** [comment]
For more context. In https://github.com/necolas/react-web-sdk I'd like to rely on this working:

```js
import styles from './styles.css';

styles['width:100%'];
=> <correct-selector-hash>
```

But at the moment `:`, `%`, and `#` are not handled properly when they are escaped in the CSS selector.

**Comment by necolas** [comment]
Thanks, this issue was fixed in 0.15.1 with the use of https://github.com/css-modules/css-selector-tokenizer/
